src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -26,6 +27,44 @@ def workqueue_stat_validation(request_args):
     stat_keys = ['total_jobs', 'input_lumis', 'input_events', 'input_num_files']
     return set(request_args.keys()) == set(stat_keys)
 
+def _validate_request_allowed_args(reqArgs, newReqArgs):
```

Comment:
By the way, we will need unit tests for this new function. But you might want to first deal with the other comments not to waste time implementing something that is still changing.

Reply:
I'll provide it in a separate commit, so it becomes merge-able  in the  set of unit tests commits.